refarch-gateway/README.md

```diff
@@ -22,7 +22,7 @@ Beside the default behaviour there are some special route prefixes which are han
 
 | Profile           | Description                                                                                       |
 |-------------------|---------------------------------------------------------------------------------------------------|
-| `json-logging`    | Switches logging from textual to JSON output.                                                     |
```

Comment:
To be honest I think `json-logging` sound like a better profile name to me.

Reply:
@DanielOber